ingest/src/main/java/com/microsoft/azure/kusto/ingest/result/IngestionStatusInTableDescription.java

```diff
@@ -1,52 +1,61 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-package com.microsoft.azure.kusto.ingest.result;
-
-import com.azure.data.tables.TableClient;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-import java.io.Serializable;
-
-public class IngestionStatusInTableDescription implements Serializable {
-    @JsonProperty
-    private String tableConnectionString;
-
-    @JsonProperty
-    private String partitionKey;
-
-    @JsonProperty
-    private String rowKey;
-
-    @JsonIgnore
-    private TableClient tableClient;
-
-    public String getPartitionKey() {
-        return partitionKey;
-    }
-
-    public void setTableConnectionString(String tableConnectionString) {
-        this.tableConnectionString = tableConnectionString;
-    }
-
-    public void setPartitionKey(String partitionKey) {
-        this.partitionKey = partitionKey;
-    }
-
-    public String getRowKey() {
-        return rowKey;
-    }
-
-    public void setRowKey(String rowKey) {
-        this.rowKey = rowKey;
-    }
-
-    public TableClient getTableClient() {
-        return tableClient;
-    }
-
-    public void setTableClient(TableClient tableClient) {
-        this.tableClient = tableClient;
-    }
-}
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.kusto.ingest.result;
+
+import com.azure.data.tables.TableClient;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.microsoft.azure.kusto.ingest.utils.TableWithSas;
+
+import java.io.Serializable;
+import java.net.URISyntaxException;
+
+public class IngestionStatusInTableDescription implements Serializable {
+    @JsonProperty
+    private String tableConnectionString;
+
+    @JsonProperty
+    private String partitionKey;
+
+    @JsonProperty
+    private String rowKey;
+
+    @JsonIgnore
+    private transient TableClient tableClient;
+

```

Comment:
Do you need both jsonignore and transient? geniunely asking, I'm not sure how this works

Reply:
as per my understanding, Looks like transient can be enough.  transient will ignore for serialization and get the default value at time of deserialization. which basically completes the functionality of jsonignore